h2/src/main/org/h2/expression/function/ToChar.java

```diff
@@ -835,9 +835,27 @@ public static String toCharDateTime(Session session, Value value, String format,
             } else if (containsAt(format, i, "TZD") != null) {
                 output.append(getTimeZone(session, value, true));
                 i += 3;
+            } else if (containsAt(format, i, "TZH") != null) {
+                int hours = DateTimeFunction.extractDateTime(session, value, DateTimeFunction.TIMEZONE_HOUR);
+                output.append( hours < 0 ? '-' : '+');
+                hours = Math.abs(hours);
+                if (hours == 0) {
+                    output.append("00");
+                } else {
+                  StringUtils.appendTwoDigits(output, hours);
+                }
```

Comment:
BTW, what is the point of this condition? Unconditional `StringUtils.appendTwoDigits(output, hours)` should produce the same result, or I'm missing something?

Reply:
There's no difference, I've merged that chance